cmd/vocli/commands/account.go

```diff
@@ -0,0 +1,54 @@
+package commands
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/spf13/cobra"
+	"go.vocdoni.io/dvote/client"
+)
+
+var accInfoCmd = &cobra.Command{
+	Use:   "info",
+	Short: "Get information about an account from the vochain. The address may or may not include the 0x prefix",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		c, err := client.New(gatewayRpc)
+		if err != nil {
+			return err
+		}
+		resp, err := c.GetAccount(nil, common.HexToAddress(args[0]))
+		if err != nil {
+			return err
+		}
+		fmt.Fprintln(Stdout, "resp", resp.String(), "err", err)
+		return err
+	},
+}
+
+var accSetInfoCmd = &cobra.Command{
+	Use:   "set <keystore, ipfs://info-URI>",
```

Comment:
`ipfs://` prefix is not required, but optional, you can see data origins on the docs

Reply:
`The URI must be http(s):// or ipfs:// prefixed and should have a maximum length (256 bytes?)` ?
from Ethereum-less-vochain-description